internal/pkg/manifest/manifesttype/type.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I wonder what's the cons of just having `internal/pkg/manifest/type.go` ? instead of having a new package

Reply:
it's a dependency problem. Mainly for mft depending on `config`